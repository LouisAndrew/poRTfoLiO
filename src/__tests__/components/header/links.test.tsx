
import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

// import { render, cleanup } from '@testing-library/react'
import { cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'

import Links  from 'components/header/links'

describe('Links', () => {
    const Element = <Links />

    afterEach(cleanup)

    it('renders without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(Element, div)
	})
	
	/* it('renders correctly', () => {
		const { getByTestId } = render()
	}) */

	it('matches snapshot', () => {
		const run = false
	    
		expect(run).toBeTruthy()

        if (run) {
	        const tree = renderer.create(Element).toJSON()
	        expect(tree).toMatchSnapshot()
	    }
	})
})
