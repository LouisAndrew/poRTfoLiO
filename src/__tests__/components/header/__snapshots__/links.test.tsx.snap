// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links matches snapshot 1`] = `
<nav
  className="css-d2po4m"
  id="links"
>
  <a
    className="css-vurnku"
    href="/#hero"
  >
    HOME
  </a>
  <a
    className="css-vurnku"
    href="/#projects"
  >
    MY PROJECTS
  </a>
  <a
    className="css-vurnku"
    href="/#about"
  >
    ABOUT ME
  </a>
  <a
    className="css-vurnku"
    href="/#contact"
  >
    CONTACT
  </a>
</nav>
`;
